src/analog/commands/uplink/getChannelsTypes.ts

```diff
@@ -22,25 +22,36 @@
 
 import * as types from '../../../types.js';
 import * as command from '../../utils/command.js';
+import channelsNames from '../../constants/channelsNames.js';
 
 
 export const id: types.TCommandId = 0x331f;
 export const name: types.TCommandName = 'getChannelsTypes';
 export const headerSize = 3;
 
 
-interface IChannelsMap {
-    channelsMap: Array<types.TUint8>
+interface IChannel {
+    type: types.TUint8,
+    name?: string
```

Comment:
```suggestion
    typeName?: string
```

like in IChannelStatus?

Reply:
fixed